packages/uxpin-merge-cli/src/utils/fs/PresetFileGenerator/PresetFileGenerator.ts

```diff
@@ -113,10 +113,24 @@ export class PresetFileGenerator {
     return !!this.componentProperties.find(isChildrenProp);
   }
 
-  private generateImports():string {
+  private async hasDefaultExport():Promise<boolean> {
+    try {
+      const file:string = await readFile(this.componentPath, 'utf8');
+      return file.indexOf('export default') !== -1;
+    } catch (e) {
+      return false;
+    }
+  }
+
+  private async generateImports():Promise<string> {
+    const relativeComponentPath:string = relative(this.presetDirectory, this.componentPath)
+      .split('.')
+      .slice(0, -1)
+      .join('.');
+    const importName:string = (await this.hasDefaultExport()) ? this.componentName : `{ ${this.componentName} }`;
```

Comment:
@robhil 

Hmm, looks like that `hasDefaultExport()` doesn't catch components which export like this: `export { Button as default };`. For example the [Button component](https://github.com/uxpin-merge/uxpin-merge-boilerplate/blob/master/src/components/Button/Button.js) of the boilerplate. In this case, the generated preset file now looks like this: `import { Button } from '../Button'`, which causes an error when trying to launch experimental mode:

```
ERROR: Error: Unknown component!
    at Object../src/components/Button/presets/0-default.jsx (/home/matthew/Documents/UXPin/Development/test/boilerplate/uxpin-merge-boilerplate/.uxpin-merge/webpack:/src/co
mponents/Button/presets/0-default.jsx:5:3)
    at __webpack_require__ (/home/matthew/Documents/UXPin/Development/test/boilerplate/uxpin-merge-boilerplate/.uxpin-merge/webpack:/webpack/bootstrap:19:1)
    at Object../.uxpin-merge/presets-70e9b712-e569-41c9-b377-bb7962c6b94f.js (/home/matthew/Documents/UXPin/Development/test/boilerplate/uxpin-merge-boilerplate/.uxpin-merg
e/__bundle__presets-70e9b712-e569-41c9-b377-bb7962c6b94f.js:197:102)
    at __webpack_require__ (/home/matthew/Documents/UXPin/Development/test/boilerplate/uxpin-merge-boilerplate/.uxpin-merge/webpack:/webpack/bootstrap:19:1)
```

Manually removing the curly braces fixes the error.